Library/Formula/wdt.rb

```diff
@@ -0,0 +1,74 @@
+class Wdt < Formula
+  desc "Transfer data as fast as possible over multiple TCP paths."
+  homepage "https://github.com/facebook/wdt"
+
+  stable do
+    url "https://github.com/facebook/wdt/archive/v1.26.1603040.tar.gz"
+    sha256 "b0fffdd12478ef74de50852b9c356352dae5f7ccee22dbca91a88afc6c5b6fee"
+
+    resource "folly" do
+      url "https://github.com/facebook/folly/archive/v0.57.0.tar.gz"
+      sha256 "92fc421e5ea4283e3c515d6062cb1b7ef21965621544f4f85a2251455e034e4b"
+    end
+
+    patch :DATA
```

Comment:
We need the patch to be submitted (and probably accepted) before we can include this. We also need the upstream submission and link and comment.